apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -49,9 +49,10 @@ namespace MWWorld
                 MWRender::RenderingManager* rendering, MWPhysics::PhysicsSystem* physics);
 
         /// If caster is an actor, the actor's facing orientation is used. Otherwise fallbackDirection is used.
-        void launchMagicBolt (const std::string& model, const std::string &sound, const std::string &spellId,
-                                     float speed, bool stack, const ESM::EffectList& effects,
-                                       const MWWorld::Ptr& caster, const std::string& sourceName, const osg::Vec3f& fallbackDirection);
+        void launchMagicBolt (const std::string &spellId, float speed, bool stack, const ESM::EffectList& effects,
+                              const MWWorld::Ptr& caster, const std::string& sourceName, const osg::Vec3f& fallbackDirection);
+
+        void ProjectileManager::getMagicBoltIDsAndSounds(std::vector<std::string>& projectileIDs, std::vector<std::string>& sounds, const ESM::EffectList& effects);
```

Comment:
The function isn't being accessed from the outside, nor does it use any state of the ProjectileManager object, so we could put it in an anonymous namespace in projectilemanager.cpp .